local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Hmm, I guess that's true. Good point @dcgoss - that would probably be even better!